numpy/testing/tests/test_utils.py

```diff
@@ -263,21 +263,22 @@ def test_complex(self):
         self._assert_func(x, x)
         self._test_not_equal(x, y)
 
-    @dec.knownfailureif(sys.version_info < (3, 0),
-                        msg="Error message different on Python 2")
     def test_error_message(self):
         try:
             self._assert_func(np.array([1, 2]), np.matrix([1, 2]))
         except AssertionError as e:
-            self.assertEqual(
-                str(e),
-                "\nArrays are not equal\n\n"
+            msg = str(e)
+            msg2 = msg.replace("shapes (2,), (1, 2)", "shapes (2L,), (1L, 2L)")
```

Comment:
Arguments the other way around.
Also, the try: except: isn't necessary if you do replacement in str(e).

Reply:
I was not thinking here; I copied the replace statement directly from the comment without looking at it myself.